.tool-versions

```diff
@@ -0,0 +1,2 @@
+java adoptopenjdk-17.0.5+8
+maven 3.8.5
```

Comment:
See my upper comment. With java image we should have these tools already...
I don't see the need for using asdf and a custom builder.

Reply:
This is a requirement of factory team, the custom builder team, as explained in the doc https://github.com/Talend/tsbi-images/tree/main/tsbi/custom-builder, is intended to be the only image used on CI and should contain everything needed to build our products and run our automation scripts.